gmakefile.test

```diff
@@ -263,19 +263,23 @@ else ifdef searchin
 else ifdef argsearch
   TESTTARGETS := $(foreach v,$(alltesttargets),$(if $(findstring $(argsearch),$($(v)_ARGS)),$(v)))
 else ifdef globsearch
+  # PETSc uses
+  # TESTTARGETS := $(shell $(PYTHON) config/query_tests.py 'name' '$(globsearch)')
+  # since v3.13.1 but it is currently not working for us
```

Comment:
```suggestion
  # TODO v3.14
  # TESTTARGETS := $(shell $(PYTHON) config/query_tests.py --testdir=$(TESTDIR) --srcdir=$(TESTSRCDIR) 'name' '$(globsearch)')
```
the original doesn't work only with a large number of tests (not our case) - we will switch to the update query_tests with 3.14

Reply:
I meant it just didn't work but that was probably because of the missing `--testdir` and `--srcdir `- will try again.

Reply:
If it is working like this with PETSc v3.13.1, we should already use this way. I don't think it's worth the TODO (and liability). I think the user who is interested in running tests will definitely be able to get the latest version.

Reply:
OK, the new form works good with v3.13.1, so I pushed it.